manifests/puppet.pp

```diff
@@ -35,10 +35,9 @@
 
   if $deploy {
     file { "${certs::pki_dir}/puppet":
-      ensure  => directory,
-      owner   => 'puppet',
-      mode    => '0700',
-      require => Class['puppet::server::install'],
```

Comment:
We could let our puppet module manage the puppet user (possibly in ```puppet::server::config```). Then puppet autorequires would fix this for us.